security/calico-network-policy.md

```diff
@@ -97,6 +97,12 @@ The Kubernetes network policy specification defines the following behavior:
 
 For compatibility with Kubernetes, **Calico network policy** follows the same behavior for Kubernetes pods.  For other endpoint types (VMs, host interfaces), Calico network policy is default deny. That is, only traffic specifically allowed by network policy is allowed, even if no network policies apply to the endpoint.
 
+### Before you begin
+
+Calicoctl must be **installed** and **configured** before use. If calicoctl is printing *"no etcd endpoints specified"*, please configure Calico to either a Kubernetes or etcd datastore.
+
+- [Install Calicoctl](https://docs.projectcalico.org/v3.5/usage/calicoctl/install)
```

Comment:
For links to our own site, we need to use the site templating engine. Because we release new docs with every release, this allows us to build relative links that don't require a massive update of the site. There are several examples of how to do this within this file that you can emulate.

Reply:
Fixed.